x/concentrated-liquidity/pool.go

```diff
@@ -253,6 +253,29 @@ func (k Keeper) GetSerializedPools(ctx sdk.Context, pagination *query.PageReques
 	return anys, pageRes, err
 }
 
+// DecreaseConcentratedPoolTickSpacing decreases the tick spacing of the given pools to the given tick spacings.
+// It returns an error if the tick spacing is not one of the authorized tick spacings or is not less than the current tick spacing of the respective pool.
+func (k Keeper) DecreaseConcentratedPoolTickSpacing(ctx sdk.Context, poolIdToTickSpacingRecord []types.PoolIdToTickSpacingRecord) error {
```

Comment:
Just to clarify, decreasing tick spacing is equivalent to increasing the number of valid ticks right? Ideally we can make this explicit in the godoc

Reply:
added a clarifying comment here https://github.com/osmosis-labs/osmosis/pull/4982/commits/87cfdd1928c4edb67bec6393fe1d10422a096958